packages/fast-tooling-react/src/css-editor/border/border.tsx

```diff
@@ -0,0 +1,129 @@
+                border: `${this.state.borderColor} ${this.state.borderStyle} ${
+                    this.state.borderWidth
+                }`.trim(),
+            });
+        }
+    }
+
+    public render(): React.ReactNode {
+        return (
+            <div className={get(this.props, "managedClasses.cssBorder")}>
+                <label className={get(this.props, "managedClasses.cssBorder_label")}>
+                    Border
+                </label>
+                <div className={get(this.props, "managedClasses.cssBorder_control")}>
+                    <div
+                        className={get(
+                            this.props,
+                            "managedClasses.cssBorder_colorInputRegion"
+                        )}
+                        style={{ background: this.state.borderColor }}
+                    >
+                        <input
+                            type={"color"}
+                            style={{ opacity: 0 }}
+                            value={this.state.borderColor}
+                            onChange={this.handleBorderColorOnChange}
+                        />
+                    </div>
+                    <span
+                        className={get(
+                            this.props,
+                            "managedClasses.cssBorder_selectControl"
+                        )}
+                    >
+                        <select
+                            className={get(this.props, "managedClasses.cssBorder_select")}
+                            onChange={this.handleBorderStyleOnChange}
+                            value={this.state.borderStyle}
+                        >
+                            {Object.keys(BorderStyleValue).map(this.renderBorderOption)}
+                        </select>
+                    </span>
+                    <input
+                        className={get(this.props, "managedClasses.cssBorder_input")}
+                        type={"text"}
+                        placeholder={"medium"}
+                        value={this.state.borderWidth}
+                        onChange={this.handleBorderWidthOnChange}
+                    />
+                </div>
+            </div>
+        );
+    }
+
+    private renderBorderOption(style: string): JSX.Element {
+        return (
+            <option key={style} value={style}>
+                {style
+                    .charAt(0)
+                    .toUpperCase()
+                    .concat(style.slice(1))}
+            </option>
+        );
+    }
+
+    private handleBorderStyleOnChange = (
+        e: React.ChangeEvent<HTMLSelectElement>
+    ): void => {
+        this.setState({
+            borderStyle: e.target.value as BorderStyleValue,
+        });
+    };
+
+    private handleBorderColorOnChange = (
+        e: React.ChangeEvent<HTMLInputElement>
+    ): void => {
+        this.setState({
+            borderColor: e.target.value.toUpperCase(),
```

Comment:
This could conceivably fall out of sync if these values are changed by an outside source.

I would actually suggest ditching `borderColor` etc. for now, this can be added as a feature afterwards because you will need to identify the sequence of these values in the object.

For example:
```ts
{
    borderWidth: "5px",
    border: "1px solid black",
    borderColor: "red"
}
```

Should identify as `borderColor` as `red`, `borderWidth` as `1px` and `borderStyle` as `solid`.